config/migrations/2023/20230713123426-update-lekp-aanvraag-step-deadline.sparql

```diff
@@ -0,0 +1,24 @@
+PREFIX m8g: <http://data.europa.eu/m8g/>
+
+DELETE {
+  GRAPH ?g {
+    ?s m8g:startTime ?start .
+    ?s m8g:endTime ?end .
+  }
+}
+INSERT {
+  GRAPH ?g {
+    ?s m8g:startTime "2023-09-01T23:59:59Z"^^xsd:dateTime .
```

Comment:
You need to take into account CEST. wich is UTC + 2 . So need to substract two hours